server/views/partials/analytics/chartBaseStackedPercentageBars.njk

```diff
@@ -1,15 +1,46 @@
-{% extends "./chartBase.njk" %}
+{% from "./chartBase.njk" import chartBase %}
 
-{#
-Required variables (in addition to base template):
-- labelColumn
-- wideLabel
-#}
+{% macro valueColumn(column, columnIndex) %}
+  {{- column -}}
+{% endmacro %}
 
-{% block chart %}
+{% macro rowTotalValue(total, rowIndex) %}
+  {{- total | thousands -}}
+{% endmacro %}
+
+{% macro rowValue(value, column, columnIndex) %}
+  {{- value | thousands -}}
+{% endmacro %}
+
+{% macro chartBaseStackedPercentageBars(
+  title,
+  guidance,
+  labelColumn,
+  graphId,
+  graphGoogleAnalyticsCategory,
+  activeCaseload,
+  form,
+  csrfToken,
+  report,
+  wideLabel=false,
+  valueColumn=valueColumn,
+  rowTotalValue=rowTotalValue,
+  rowValue=rowValue
+) %}
 
 {% set colourChoices = report.columns | chartPalette %}
 
+{% call chartBase(
+  title=title,
+  guidance=guidance,
+  graphId=graphId,
+  graphGoogleAnalyticsCategory=graphGoogleAnalyticsCategory,
+  activeCaseload=activeCaseload,
+  form=form,
+  csrfToken=csrfToken,
+  report=report
+) %}
+
 <div class="govuk-!-margin-top-8">
```

Comment:
I wouldn't be against indenting the blocks between `{% call ...%}` and `{% endcall %}`.

Reply:
there is a lot of indenting i want to do, but didn't in this PR cos then every line of the diff would look modified